server/server.js

```diff
@@ -5,28 +5,31 @@ const cors = require('cors');
 const app = express();
 const port = 3000;
 const dumbData = require('../dumbData');
+const data = require('../fakeData')
 
 app.use(bodyParser.urlencoded({extended: true}));
 app.use(cors());
 
 app.use(express.static('../client/public'))
 
-app.get('/test', (req, res) => {
-    if (!dumbData.simpleData){
+
+app.get('/listing', (req, res) => {
+    if (!data.fakeData){
         res.status(404).json({message: 'no data here'})
     }
-    res.json(dumbData.simpleData);
+    res.json(data.fakeData);
 });
 
-app.get('/test/:id', (req, res) => {
+app.get('/listing/:id', (req, res) => {
     const requestId = req.params.id;
 
-    let item = dumbData.simpleData.filter((question) => {
-        return requestId == question.id;
+    let item = data.fakeData.filter((questions) => {
```

Comment:
Confusing because ```questions``` in this line refers to a single ```listingQuestionsAndAnswers``` in ```fakeData.js```. Again, would rename to ```listing``` or ```attraction```